README.md

```diff
@@ -36,3 +36,12 @@ the rewritten target URL might look like this:
 
 and the text 'About the REDCap Mobile App' will be added on the 'REDCap Mobile App' before the title 'What is the REDCap Mobile App?'.
 
+
+## Module Workflow
+1. The user requests a token: On the project in question, the user clicks on the left menu link **REDCap Mobile App** and then on the button **Request API token**. He is redirected to the URL provided during the configuration of the module. Once he completes the questionnaire, the token request will be submitted.
+2. The project manager/administrator receives the request.
+3. The project manager/administrator approves the request.
+4. The project manager/administrator disables the module: The project manager/administrator disables the module in the project by going to the ** External Modules** page (under Applications) and clicking **Disable** next to the **Route Mobile Api Token Requests to Survey** module.
```

Comment:
we must be clear about where the module disable event occurs.  Maybe the title for this bullet should read `The project manager/administrator disables the module at the project level`.  Then add emphasis in the sentence.  Maybe like this:

```
The project manager/administrator disables the module level: The project manager/administrator disables the module in the _project_ by going to the **External Modules** page (under Applications) and clicking **Disable** next to the **Route Mobile Api Token Requests to Survey** module.
```